inspirehep/modules/records/serializers/fields_export.py

```diff
@@ -63,6 +63,23 @@ def decorator(decorated_function):
     return extractor
 
 
+def is_phd_thesis(obj):
+    """Determine if a record is a PhD thesis
+
+    First checks thesis_info then public_notes to determine if a record
+    is a PhD thesis.
+    """
+    if get_value(obj, 'thesis_info.degree_type') in ('phd', 'habilitation'):
+        return True
+
+    public_notes = get_value(obj, 'public_notes.value', [])
+    for note in public_notes:
+        if "phd thesis" in note.lower():
+            return True
```

Comment:
@michamos do we want this?

Reply:
no, reading `thesis_info.degree_type` is sufficient. If it so happens that many theses don't have that field but contain `phd thesis` in `public_notes`, we need to clean up the metadata.

Reply:
Seems I was unlucky: first record from arXiv I checked had this issue, but looking at more now, it doesn't seem all that common. I'll remove this then.

Reply:
anyway, when metadata is not where you expect it, better to ask :)